src/wkwebkit.cs

```diff
@@ -856,11 +865,13 @@ interface WKWebView
 		SecTrust ServerTrust { get; }
 
 #if !MONOMAC
+		[NoMac][MacCatalyst (13,1)]
 		[iOS (11,0)]
 		[Async]
 		[Export ("takeSnapshotWithConfiguration:completionHandler:")]
 		void TakeSnapshot ([NullAllowed] WKSnapshotConfiguration snapshotConfiguration, Action<UIImage, NSError> completionHandler);
 #else
+		[NoiOS][NoMacCatalyst]
```

Comment:
You can merge the two `TakeSnapshot` definitions (and remove the `#if ...` condition), because you've defined UIImage to be NSImage on macOS (at the top of the file, with the `using` directives):

```cs
 		[Mac (10,13), iOS (11,0)]
 		[MacCatalyst (13,1)]
 		[Async]
 		[Export ("takeSnapshotWithConfiguration:completionHandler:")]
 		void TakeSnapshot ([NullAllowed] WKSnapshotConfiguration snapshotConfiguration, Action<UIImage, NSError> completionHandler);
```